src/luos.c

```diff
@@ -45,9 +45,9 @@ static int luos_msg_handler(module_t* module, msg_t* input, msg_t* output) {
         output->header.cmd = LUOS_REVISION;
         output->header.target_mode = ID;
 #ifndef LUOS_REV
-#define LUOS_REV "unknown"
```

Comment:
No we should keep the unknown!
If the revision don't come from somewhere the revision is unknown.